script-library/homebrew-debian.sh

```diff
@@ -117,12 +117,12 @@ if [ "${USE_SHALLOW_CLONE}" = "false" ]; then
     echo "Installing Homebrew..."
     git clone https://github.com/Homebrew/brew "${BREW_PREFIX}/Homebrew"
     mkdir -p "${BREW_PREFIX}/Homebrew/Library/Taps/homebrew"
-    git clone https://github.com/Homebrew/linuxbrew-core "${BREW_PREFIX}/Homebrew/Library/Taps/homebrew/homebrew-core"
+    git clone https://github.com/Homebrew/homebrew-core "${BREW_PREFIX}/Homebrew/Library/Taps/homebrew/homebrew-core"
```

Comment:
Thanks for contributing! Why did you make this change?

Reply:
> Thanks for contributing! Why did you make this change?

Previously, Homebrew used a separate repo for Linux, but now they merged Mac and Linux repos into a single one: https://github.com/Homebrew/linuxbrew-core/blob/master/README.md

Reply:
gotcha, thanks.